base_restrict_user_fields/security/restrict_user_fields.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="group_restrict_fields" model="res.groups">
```

Comment:
As group indicates the opposite meaning, please rename it (and added context, because ID should be ambiguous if not):
```suggestion
        <record id="group_user_mgmt_unrestrict_fields" model="res.groups">
```